README.md

```diff
@@ -196,6 +197,60 @@ await client.time()
 await client.close()
 ```
 
+### Using the Realtime API
+The python realtime API currently only supports authentication with ably API key.
+#### Creating a client
+```python
+from ably import AblyRealtime
+
+async def main():
+    client = AblyRealtime('api:key')
+    channel = client.channels.get('channel_name)
+```
+
+#### Subscribing to a channel for event
+```python
+message_future = asyncio.Future()
```

Comment:
I think you should remove the `message_future` bit because that's a little confusing. Just change it so that the listener prints out the `message.data`.